lms/models/solutions.py

```diff
@@ -6,13 +6,50 @@
 
 from flask_babel import gettext as _
 
-from lms.lmsdb.models import Solution, SolutionFile
+from lms.lmsdb.models import Comment, Solution, SolutionFile, User
 from lms.lmstests.public.general import tasks as general_tasks
 from lms.lmstests.public.identical_tests import tasks as identical_tests_tasks
 from lms.lmsweb import config, routes
 from lms.models import notifications
 
 
+def send_notification_after_check(user: User, solution: Solution) -> bool:
+    is_checked = solution.is_checked
+    if is_checked:
+        if solution.solver == user:
+            msg = _(
+                '%(solver)s הגיב לך על בדיקת תרגיל "%(subject)s".',
+                solver=solution.solver.fullname,
+                subject=solution.exercise.subject,
+            )
+            addressee = solution.checker
+        else:  # solution.checker == user
+            msg = _(
```

Comment:
Call the function after the if/else clause. Set a dictionary of the params inside/before the if/else.